contentcuration/contentcuration/frontend/channelEdit/vuex/task/index.js

```diff
@@ -47,14 +48,24 @@ export default {
       return Task.deleteModel(task.task_id).then(() => store.dispatch('activateTaskUpdateTimer'));
     },
     updateTaskList(store) {
-      return Task.where({ channel: store.rootState.currentChannel.currentChannelId })
-        .then(tasks => {
-          store.commit('SET_ASYNC_TASKS', tasks);
-          store.dispatch('activateTaskUpdateTimer');
-        })
-        .catch(() => {
-          store.dispatch('activateTaskUpdateTimer');
-        });
+      if (store.rootGetters.loggedIn) {
+        return Task.where({ channel: store.rootState.currentChannel.currentChannelId })
+          .then(tasks => {
+            store.commit('SET_ASYNC_TASKS', tasks);
+            store.dispatch('activateTaskUpdateTimer');
+          })
+          .catch(e => {
+            // if not authorized, redirect to login screen
+            if (e.response.status === 403) {
+              this.$router.push('/');
```

Comment:
Not really sure what Vue Router does in this instance - but maybe it's more explicit to call the same as below here?
`window.location.href = urls.login();`

And maybe returning within the `if` block just to be explicit that nothing else should happen?